src/index.js

```diff
@@ -0,0 +1,71 @@
+/** Copyright (c) 2017 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const requiredLabels = [
+  'breaking',
+  'feature',
+  'bugfix',
+  'docs',
+  'discussion',
+  'release',
+  'prerelease',
+  'greenkeeping',
+];
```

Comment:
- This perhaps this could be made generic, sort of like https://github.com/uber-workflow/probot-app-release-notes/blob/01404d25552ba2f57f80ed0d8258dab9b15c36ea/index.js#L166-L176 so it doesn't have to be hardcoded. Ideally the label-related coupling exists as config in the composite host probot instance rather than implicit coupling w/ the individual sub-probots themselves.